### PR TITLE
strange seeds have their own crate now, other seed crate prices lowered

### PR DESCRIPTION
**Pull request by Commandersand**
## Why It's Good For The Game
To get strange seeds you need to get a pack with a lot of seeds you probably don't need
This helps stop clutter on the shuttle and at cargo

Oh yeah i lowered the other crate costs because some of those seeds are round start free in public areas and it's hard to run out of them

## Changelog
:cl:
add: You can now order dedicated strange seed crates
add: Regular and exotic crate costs lowered
/:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
I think your missing the point if exotic where it can be any mutation seeds too

**Comment by CinderWC** [comment]
I dont particularly think that this needs any change whatsoever as botany is already a **VERY STRONG** department when used in the right hands. I consistently see them churn out products like a goddamn factory and this will only serve to make it easier for them.

Hard no from me

**Comment by improvedname** [comment]
I agree with cinder here strange seeds allow you to get any chem in the game including the almighty adminorizine.

Though you could port a crate from TG which sells strange seeds for x money but this price reducements is ridicilous

**Comment by improvedname** [comment]
Also it isnt modularized

File: code/modules/cargo/packs/organic.dm [hunk 1]
```diff
@@ -329,7 +329,7 @@
 /datum/supply_pack/organic/seeds
 	name = "Seeds Crate"
 	desc = "Big things have small beginnings. Contains fourteen different seeds."
-	cost = 1250
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs/organic.dm:332]: Should lable this all, cuz ~~im~~ people upstream are going to be tweaking these

File: code/modules/cargo/packs/organic.dm [hunk 2]
```diff
@@ -360,11 +360,23 @@
...
 					/obj/item/seeds/banana,
-					/obj/item/seeds/eggplant/eggy,
+					/obj/item/seeds/eggplant/eggy)
+	crate_name = "exotic seeds crate"
+	crate_type = /obj/structure/closet/crate/hydroponics
+	
+	/datum/supply_pack/organic/strangeseeds //skyrat addition
+	name = "Seeds Crate (Strange)"
+	desc = "A crate of strange seeds, contains 6 seeds found on alien planets"
+	cost = 1500
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs/organic.dm:370]: These are massivly powerfull and just for around 800? Thats way way to cheap.

**Comment by Commandersand** [comment]
upped to 5000

**Comment by improvedname** [comment]
Still not a reason to lower other seed crates
Also not modular enough you have to make its its own file in modular_skyrat

**Comment by CinderWC** [comment]
This does not change the fact that it makes Botany easier than it should be, i am still fully against this

**Comment by improvedname** [comment]
The strange seed crate is fine there is a small chance to be auctally usefull and its gonna be a heavy money investation of cargo if they accept it that is

File: code/modules/cargo/packs/organic.dm [hunk 3]
```diff
@@ -350,7 +350,7 @@
 /datum/supply_pack/organic/exoticseeds
 	name = "Seeds Crate (Exotic)"
 	desc = "Any entrepreneuring botanist's dream. Contains twelve different seeds, including three replica-pod seeds and two mystery seeds!"
-	cost = 1500
+	cost = 1200 //skyrat change
```
Comment (CinderWC) [inline code/modules/cargo/packs/organic.dm:353]: spell check

File: code/modules/cargo/packs/organic.dm [hunk 4]
```diff
@@ -329,7 +329,7 @@
 /datum/supply_pack/organic/seeds
 	name = "Seeds Crate"
 	desc = "Big things have small beginnings. Contains fourteen different seeds."
-	cost = 1250
+	cost = 1000 //skyrate change
```
Comment (CinderWC) [inline code/modules/cargo/packs/organic.dm:332]: Spell check

**Comment by Commandersand** [comment]
>easier than it should be

with the new costs it does cost more to get the same amount of strange seeds but with less crates cluttering cargo or botany

**Comment by Commandersand** [comment]
bad PR also i'm lazy

**Pull request closed without merging**